DataFormats/TauReco/interface/RecoTauPiZero.h

```diff
@@ -14,32 +14,57 @@ class RecoTauPiZero : public CompositePtrCandidate {
       kStrips = 3
     };
 
-    RecoTauPiZero():CompositePtrCandidate(),algoName_(kUndefined){
-      this->setPdgId(111); }
-
-    RecoTauPiZero(PiZeroAlgorithm algoName):
-        CompositePtrCandidate(), algoName_(algoName) { this->setPdgId(111); }
+    RecoTauPiZero()
+      : CompositePtrCandidate(),
+        algoName_(kUndefined)
```

Comment:
I'm afraid I still don't understand - at least, it's not compiling if I do:
    RecoTauPiZero()
      : CompositePtrCandidate(),
        algoName_(kUndefined), bendCorrEta_, bendCorrPhi_
    {
      bendCorrEta_ = 0.;
      bendCorrPhi_ = 0.;
      this->setPdgId(111);
    }


or if I do:

    RecoTauPiZero()
      : CompositePtrCandidate(),
        algoName_(kUndefined), bendCorrEta_ = 0., bendCorrPhi_ = 0.
    {
      this->setPdgId(111);
    }